upstart_msh/upstart_install.sh

```diff
@@ -1,28 +1,45 @@
 #!/bin/bash
 
+set -e
+
 robot_name="${HOSTNAME//-b1}"
+ROS_DISTRO="indigo"
+
+sudo cp -f /u/robot/git/setup_cob4/upstart_msh/cob.conf /etc/init/cob.conf
+sudo cp -f /u/robot/git/setup_cob4/upstart_msh/cob_msh.conf /etc/init/cob_msh.conf
 
-sudo cp /u/robot/git/setup_cob4/upstart_msh/cob.conf /etc/init/cob.conf
-sudo cp /u/robot/git/setup_cob4/upstart_msh/cob_msh.conf /etc/init/cob_msh.conf
-sudo cp /u/robot/git/setup_cob4/upstart_msh/cob-start /usr/sbin/cob-start
+sudo cp -f /u/robot/git/setup_cob4/upstart_msh/cob-start /usr/sbin/cob-start
 sudo sed -i "s/myrobotname/$robot_name/g" /usr/sbin/cob-start
 sudo sed -i "s/mydistro/$ROS_DISTRO/g" /usr/sbin/cob-start
 sudo sed -i "s/myrobot/$ROBOT/g" /usr/sbin/cob-start
 sudo sed -i "s/myuser/msh/g" /usr/sbin/cob-start
-echo "%users ALL=NOPASSWD:/usr/sbin/cob-start" | sudo tee -a /etc/sudoers
+sudoers_string="%users ALL=NOPASSWD:/usr/sbin/cob-start"
```

Comment:
please be consequence, upstart/install_upstart has other command line for the same function : https://github.com/ipa-nhg/setup_cob4/blob/042f712e572bceaa12ff32ffe9ed93fc31b49367/upstart/upstart_install.sh#L14 , modify both or use here the same command line as upstart_install.sh